config/vector.spec.toml

```diff
@@ -1535,6 +1535,82 @@ dns_servers = ["0.0.0.0:53"]
   # * type: string
   value = "/var/log/nginx.log"
 
+# Accepts and outputs `log` and `metric` events allowing you to select events based on a set of logical conditions.
+[transforms.filter]
+  #
+  # General
+  #
+
+  # The component type. This is a required field that tells Vector which
+  # component to use. The value _must_ be `#{name}`.
+  #
+  # * required
+  # * type: string
+  # * must be: "filter"
+  type = "filter"
+
+  # A list of upstream source or transform IDs. See configuration for more info.
+  #
+  # * required
+  # * type: [string]
+  inputs = ["my-source-id"]
+
+  #
+  # Condition
+  #
+
+  [transforms.filter.condition]
+    # The type of the condition to execute.
+    #
+    # * optional
+    # * default: "check_fields"
+    # * type: string
+    # * enum: "check_fields", "is_log", and "is_metric"
+    type = "check_fields"
+    type = "is_log"
+    type = "is_metric"
+
+    # Check whether a fields contents exactly matches the value specified.
+    #
+    # * optional
+    # * no default
+    # * type: string
+    # * relevant when type = "check_fields"
+    "message.eq" = "this is the content to match against"
+
+    # Check whether a field exists or does not exist, depending on the provided
+    # valuebeing `true` or `false` respectively.
```

Comment:
```suggestion
    # value being `true` or `false` respectively.
```

Reply:
Well spotted! Odd, though, that that's been in the TOML since it was first written (d5ef3cdf0c1cdf3bcc660440ac8a52fae0824309) and only now changed in the output.

Reply:
So odd!